.github/srvaroa_labeler.yml

```
@@ -41,18 +41,30 @@ labels:
+    below: 400
 
 - label: "Size: Big"
   size:
-    above: 400
+      above: 399
+      below: 800
 
 - label: "Size: Huge"
   size:
-    above: 800
+      above: 799
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 6

(indentation)